bower.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Winstrap",
-    "version": "0.5.10",
+    "version": "0.5.11",
```

Comment:
Please don't change this yet (and the one in package.json). This should be updated right when a new version is ready to be released.